android/KMEA/build.sh

```diff
@@ -106,6 +105,7 @@ done
 # by developer. As it's not CI, the Web artifacts won't exist otherwise...
 # unless the developer manually runs the correct build configuration accordingly.
 if [[ $VERSION_ENVIRONMENT == "local" ]] && [[ $UPLOAD_SENTRY == true ]]; then
+    # TODO:  handle the -upload-sentry in its eventual new form
```

Comment:
- [ ] Resolve Sentry symbol uploading.

Reply:
iirc, we're actually not using -upload-sentry. I didn't think Sentry handles Android sourcemaps.

Reply:
@mcdurdin Hoping for confirmation on this before I outright-remove any related code.

Reply:
Correct, Sentry does not currently support Android sourcemaps.